src/Services/ListService.php

```diff
@@ -114,6 +114,21 @@ public function getListOption($list_id, $option_id)
         return null;
     }
 
+    /**
+     * Retrieves a list option for a given list by the code value
+     * @param string $list_id
+     * @param string $code The exact code(s) to match.  This must match the string precisely and fuzzy matching is not currently supported.
```

Comment:
note that can place multiple codes in the codes column (in the lists gui), so may need "fuzzy" at some point